client/my-sites/comments/comment/comment-author.jsx

```diff
@@ -71,6 +82,9 @@ export class CommentAuthor extends Component {
 
 				<div className="comment__author-info">
 					<div className="comment__author-info-element">
+						{ this.commentHasLink() && (
+							<Gridicon icon="link" size="18" className="comment__author-has-link" />
```

Comment:
Gridicons' `size` should be an actual number (e.g. `size={ 18 }`), or it'd throw a console warning.

See https://github.com/Automattic/gridicons/blob/master/sources/react/index-header.jsx#L24